src/Symfony/Bridge/Doctrine/Security/MagicLink/MagicLoginLinkTokenEntityTrait.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Security\MagicLink;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Trait that can be used by your entity to help implement StoredMagicLinkTokenInterface.
+ *
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ */
+trait MagicLoginLinkTokenEntityTrait
+{
+    /**
+     * @ORM\Column(type="string", length=20, unique=true)
+     */
+    private $selector;
+
+    /**
+     * @ORM\Column(type="string", length=100)
+     */
+    private $hashedVerifier;
```

Comment:
This terminology sounds strange to me. Could we rename it as `signature`?

Reply:
I don't know if that's cryptographically accurate. I like signature better too... but this is more of a "password". The raw "password" is included as part of the magic link that you follow. Then we grab that password from the URL, hash it, and compare its hash with this field (exactly like a password).